collector/transport_node_collector.go

```diff
@@ -54,6 +60,7 @@ func newTransportNodeCollector(transportNodeClient client.TransportNodeClient, l
 	return &transportNodeCollector{
 		transportNodeClient:   transportNodeClient,
 		logger:                logger,
+		transportNodeStatus:   transportNodeStatus,
```

Comment:
If we reintroduce transportNodeStatus desc then it would be good to put it on Describe too

Reply:
done 88df537